examples/supertuxkart/main.go

```diff
@@ -57,58 +69,118 @@ func main() {
 	cmd.Stdout = os.Stdout
 
 	if err := cmd.Start(); err != nil {
-		log.Fatalf(">>> Error starting cmd: %v", err)
+		log.Fatalf("error starting cmd: %v", err)
 	}
 
 	// SuperTuxKart refuses to output to foreground, so we're going to
 	// poll the server log.
-	ready := false
 	home, err := os.UserHomeDir()
 	if err != nil {
-		log.Fatalf(">>> Could not get home dir: %v", err)
+		log.Fatalf("could not get home dir: %v", err)
 	}
-	for i := 0; i <= 10; i++ {
-		time.Sleep(time.Second)
 
-		logFile, err := os.Open(path.Join(home, logLocation))
-		if err != nil {
-			log.Printf(">>> Could not open server log: %v", err)
-			continue
-		}
+	t := &tail.Tail{}
+	for i := 0; i < 10; i++ {
```

Comment:
If you are tailing the file why keep the sleep & loop?

Reply:
I left this in to ensure that the log file has time to get created by stk. My understanding is that tail will error if the file doesn't exist yet.